src/sentry/api/endpoints/group_details.py

```diff
@@ -160,9 +160,28 @@ def _get_release_info(self, request, group, version):
                 version=version,
             )
         except Release.DoesNotExist:
-            return {"version": version}
+            release = {"version": version}
         return serialize(release, request.user)
 
+    def _get_first_last_release_info(self, request, group, versions):
+        releases = {
+            release.version: release
+            for release in Release.objects.filter(
+                projects=group.project,
+                organization_id=group.project.organization_id,
+                version__in=versions,
+            )
+        }
+        tag_values = tagstore.get_release_tags(
+            [group.project_id], None, [version for version in versions if version in releases],
+        )
```

Comment:
Do we actually get any benefit out of doing this here? If we just pass all the releases to the one serialize call then it'll do this batching for us, and we don't need to pass the `tag_values` through.

Reply:
oops no, didn't think to pass all the releases to the serialize call. Fixed in 2e4b7e0